### DISCUSSION TITLE
浅入浅出智能合约 - 概述（一） · /smart-contract-intro

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#521] [Announcements]
https://draveness.me/smart-contract-intro

**Comment by 2722119447** [comment]
楼主你好，推荐一个以太坊dapp开发区块链应用的教程：http://xc.hubwiz.com/course/5a952991adb3847553d205d1?affid=20180507draveness.me

是否有可能合作一个课程？

**Comment by howardpen9** [comment]
這個版面小細節很棒啊! 竟然程式碼可以用黑底的，閱讀感受很好! 讚!! 可否fork一下這個blog版面?
